### PR TITLE
Add support for `eli5` OpenAI command

### PR DESCRIPTION
**Pull request by zdhickman**
Usage:

`botbot eli5 cryptocurrency`
`botbot eli5 temp:0.5 cryptocurrency`

Includes a minor refactor to split out parsing of prefix (based on command), prompt, and temp

### CODE REVIEW

File: botbot/Module/OpenAIModule.cs [hunk 1]
```diff
@@ -24,26 +25,49 @@ public OpenAIModule()
...
+                return "Explain like I'm 5 years old: ";
+            }
+
+            return "";
+        }
+
+        private string GetPrompt(string text) {
+            string[] splitText = text.Split(' ');
+            if (splitText[2].StartsWith("temp")) {
+                return text.Substring(
```
Comment (zdhickman) [inline botbot/Module/OpenAIModule.cs]: If you like how these methods are split up then I can clean this up to be less ugly

File: botbot/Module/OpenAIModule.cs [hunk 2]
```diff
@@ -24,26 +25,49 @@ public OpenAIModule()
             encoder = Encoder.Get_Encoder();
         }
 
+        private string GetPrefix(string text) {
+            string[] splitText = text.Split(' ');
+            string command = splitText[1];
+            if (String.Equals(command, "eli5") ){
```
Comment (golf1052) [inline botbot/Module/OpenAIModule.cs]: Can just do `if (command == "eli5")`, C# has operator overloading

File: botbot/Module/OpenAIModule.cs [hunk 3]
```diff
@@ -24,26 +25,49 @@ public OpenAIModule()
...
-                float temp = 1;
-                if (text.StartsWith("temp"))
-                {
-                    string[] splitText = text.Split(' ');
-                    string[] splitTemp = splitText[0].Split(':');
-                    temp = float.Parse(splitTemp[1]);
-                    text = text.Substring(splitText[0].Length).Trim();
-                }
+                string prefix = GetPrefix(text);
+                string prompt = prefix + GetPrompt(text);
```
Comment (golf1052) [inline botbot/Module/OpenAIModule.cs]: Don't really like the double string.Split here. Should split the text before and pass it to both methods.

**Review by golf1052 — CHANGES_REQUESTED** [review]

File: botbot/Module/OpenAIModule.cs (hunk 1 above)
Comment (golf1052) [reply botbot/Module/OpenAIModule.cs]: Yeah it should be formatted in the "C#" way, should be able to just use the format command in VSCode.

Comment (zdhickman) [reply botbot/Module/OpenAIModule.cs]: I'd need to use your config because my default prettier config formats different from the patterns in this repo

File: botbot/Module/OpenAIModule.cs (hunk 3 above)
Comment (zdhickman) [reply botbot/Module/OpenAIModule.cs]: Sure, I ended up reducing to a single helper because it gets messy trying to work with the partial strings in the other helper methods

**Review by golf1052 — APPROVED** [review]

File: botbot/Module/OpenAIModule.cs (hunk 1 above)
Comment (golf1052) [reply botbot/Module/OpenAIModule.cs]: I can fix it in post.
